pages/es/_syntax.md

```diff
@@ -59,22 +61,17 @@ Si una cadena contiene una o varias comillas simples, se podría considerar envo
 
 ### URLs
 
-Las URLs deben ir entre comillas simples por las mismas razones que se explican anteriormente:
+Las URLs deben ir entre comillas por las mismas razones que se explican anteriormente:
 
 {% include snippets/syntax/06/index.html %}
 
-###### Más información
-
-* [Todo lo que necesitas saber sobre la interpolación en Sass -En inglés](http://webdesign.tutsplus.com/tutorials/all-you-ever-need-to-know-about-sass-interpolation--cms-21375)
-* [SassyStrings -En inglés](https://github.com/HugoGiraudel/SassyStrings)
-
 ## Números
 
-En Sass, el número es un tipo de datos que incluye de todo, desde números sin unidades a longitudes, pasando por duraciones, frecuencias y ángulos, entre otros. Esto permite que se puedan realizar cálculos en estas medidas.
+En Sass, número es un tipo de datos que incluye de todo, desde números sin unidades a longitudes, pasando por duraciones, frecuencias y ángulos, entre otros. Esto permite que se ejecuten cálculos sobre tales medidas.
 
 ### Ceros
 
-Siempre se deben mostrar los ceros a la izquierda antes de un valor decimal menor que uno. Nunca mostrar ceros finales.
+Siempre se deben mostrar los ceros a la izquierda antes de un valor decimal menor que uno. Nunca mostrar los ceros finales.
 
```

Comment:
There is a paragraph missing here. See English version for reference: https://github.com/HugoGiraudel/sass-guidelines/pull/223/files#diff-8fb504ae90cfe32f3ba366da9f47f1ddR85.